src/gt4py/stencil_object.py

```diff
@@ -396,6 +395,8 @@ def _validate_args(  # noqa: C901  # Function is too complex
                 f"Compute domain too small. Sequential axis is {domain[2]}, but must be at least {self.domain_info.min_sequential_axis_size}."
             )
 
+        gt_backend = gt4py.backend.from_name(self.backend)
```

Comment:
In order not to confuse this with past code that uses `gt_backend` to refer to `gt4py.backend`, I will rename this to `stencil_backend`.